github/teams.go

```diff
@@ -203,6 +203,52 @@ func (s *TeamsService) EditTeam(ctx context.Context, id int64, team NewTeam) (*T
 	return t, resp, nil
 }
 
+// RemoveParentTeam represents a new team with no parent
```

Comment:
Comments should be complete sentences ended in a period for the auto-generated godocs.

Reply:
Removing duplicate function